### PR TITLE
Login

### PR DESCRIPTION
**Pull request by tomas-schmidt**
## Summary

- Add login screen and interaction with API, if login is successful access_token is logged.

## Screenshots

image

## Trello Card

- https://trello.com/c/MCb1boXh/13-login

### CODE REVIEW

File: wolox-books/src/app/screens/login/login.component.ts [hunk 1]
```diff
@@ -1,15 +1,35 @@
...
+      'password': [null, Validators.required]
+    }
+  }
 
-  ngOnInit() {
+  loginUser() {
+    this.userService.loginUser({ session: this.loginForm.value })
+    .subscribe(
+      resp => {
+        console.log(resp['access_token']);
```
[the remark this answers is not in the document]
Comment (tomas-schmidt) [reply wolox-books/src/app/screens/login/login.component.ts]: Next card task

File: wolox-books/src/scss/forms.scss [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+.form-container {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  width: $form-width;
+  background-color: $grey;
+  border-top: 5px solid $blue;
+  padding: 20px;
+  margin: auto;
```
[the remark this answers is not in the document]
Comment (tomas-schmidt) [reply wolox-books/src/scss/forms.scss]: Margin auto centers the form

File: wolox-books/src/scss/forms.scss [hunk 3]
```diff
@@ -0,0 +1,33 @@
+$form-width: 350px;
+$header-logo-width: 300px;
+
+.form-container {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  width: $form-width;
```
Comment (mcallegari10) [inline wolox-books/src/scss/forms.scss]: Being the `form-width: 350px`, use `max-width` to set it to your variable, and then use `width: 100%` for it to only have it's maximum width available. If not, the layout might break on small screens 😄

File: wolox-books/src/scss/buttons.scss [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+  &:disabled {
+    border: 2px solid $green;
+    color: $green;
+    background-color: white;
+    cursor: not-allowed;
+  }
+
+  &:active {
+    box-shadow: 0 5px $green-shadow;
+    transform: translateY(5px);
```
Comment (mcallegari10) [inline wolox-books/src/scss/buttons.scss]: Why do we need this `transform`?

**Pull request merged**